fairlearn/preprocessing/_correlation_remover.py

```diff
@@ -100,3 +105,17 @@ def transform(self, X):
         X_use = np.atleast_2d(X_use)
         X_filtered = np.atleast_2d(X_filtered)
         return self.alpha * X_filtered + (1 - self.alpha) * X_use
+
+    def _more_tags(self):
+        return {
+            "_xfail_checks": {
```

Comment:
This looks like test things in the product code? I'm sure this is the right thing to do, but can you explain what's going on? Is `_more_tags()` a magic name for something in pytest?

Reply:
no it has nothing to do with pytest. Estimator tags are things which can be set on an estimator _instance_, which can define its behavior in certain ways, including when the test suite runs tests on it.

A bit more about tags here: https://scikit-learn.org/stable/developers/develop.html?highlight=estimator%20tags#estimator-tags

and this particular tag is used here in the test suite: https://github.com/scikit-learn/scikit-learn/blob/844b4be24d20fc42cc13b957374c718956a0db39/sklearn/utils/estimator_checks.py#L502

We are working on making the interface for these tags a bit better than what it is.

Reply:
Thanks for the explanation.